### ISSUE TITLE
how to format money?

### ISSUE THREAD

**Issue by vinnyparker** [crocodic-studio/crudbooster#440]
Hi, I'm studiyng crudbooster and it's awesome. thanx a lot.
So, I have a field in my form that its money. but by default it's using 3 decimals (90,320), and I need to use 2 decimals(90,32) how do I change it?
I'm new at php and crudboster, so, I'm sorry if I'm asking some noob stuff.
Sorry my engllish is not that good
![captura de tela de 2017-04-28 21-43-24](https://cloud.githubusercontent.com/assets/1207567/25551520/5a0858e2-2c5c-11e7-87cb-bcda671e46e9.png)

**Comment by davodavodavo3** [comment]
@vinnyparker hi! Are you here?

**Comment by vinnyparker** [comment]
now i do

**Comment by davodavodavo3** [comment]
#443 PR request for decimal and decimal points on money field

**Comment by vinnyparker** [comment]
thank you! so I just downlload a new copy of the main.js and subscrybe mine?

**Comment by vinnyparker** [comment]
ops, now i see that is not only this class/file. I'll download all files of your pull request and try again. 
once again, thanx

**Comment by davodavodavo3** [comment]
If all work fine please close issue. This PR will be added sonn in newer version

**Comment by vinnyparker** [comment]
I'm downloading for test right now

**Comment by vinnyparker** [comment]
Hi. I download that files, but i got this.

![captura de tela de 2017-05-01 12-51-43](https://cloud.githubusercontent.com/assets/1207567/25584573/f9b8fdc6-2e6c-11e7-89ab-28ef2ef59c4f.png)

**Comment by davodavodavo3** [comment]
show me the form[] code for fields

**Comment by vinnyparker** [comment]
you mean the controller.php?

**Comment by vinnyparker** [comment]
https://pastebin.com/PEQmDw5n

**Comment by davodavodavo3** [comment]
Hmm i dont have a error,@vinnyparker can you add 
`$(document).ready(function(){`
in money/asset.blade  script function

**Comment by davodavodavo3** [comment]
```
<script>
	$(document).ready(function(){

		$('.inputMoney').priceFormat({
			prefix: '',
			thousandsSeparator: '{!! $form['dec_point']?: '' !!}',
			centsLimit: {!! $form['decimals']?: "''" !!},
			clearOnEmpty:true,
		});

	})
</script>
```

**Comment by davodavodavo3** [comment]
Example of form if you need:
`$this->form[] = ['label'=>'Min. sum','name'=>'min_sum','type'=>'money','validation'=>'required|numeric','width'=>'col-sm-10','decimals'=>'2','dec_point'=>'','help'=>'(включительно)'];`

**Comment by vinnyparker** [comment]
well. I did it, and I'm still have the error. I even add the ,'decimals'=>'2','dec_point'=>'' to the controller, but still have an error.
I have a doubt. What is this files? because the error makes references to a file named: "FatalErrorException in 7b7f018af5e4c853d59e3bf9bb18e8f591862f79.php line 2:"

![captura de tela de 2017-05-01 14-41-45](https://cloud.githubusercontent.com/assets/1207567/25587898/7139349c-2e7c-11e7-8365-295a4a754a46.png)

**Comment by vinnyparker** [comment]
Well, now i made it work.
I undo the #443  and add to the form[]: 
`$this->form[] = ['label'=>'Valor','name'=>'valor','type'=>'money','validation'=>'required|min:0|integer','width'=>'col-sm-10','decimals'=>'2','dec_point'=>'.'];`
and then it was formated in the right way, as you can see.  at the image below. 
Once again. Thanx a lot for helping me and spend your time! thank you verymuch
![captura de tela de 2017-05-01 15-00-00](https://cloud.githubusercontent.com/assets/1207567/25588514/f933b7b2-2e7e-11e7-8ef3-34e18ab9163b.png)

**Comment by davodavodavo3** [comment]
ok , you update from #443 and all works?I did not understand undo the #443!

**Comment by vinnyparker** [comment]
I downgrade from #443  and just use the formatter on form[].

**Comment by davodavodavo3** [comment]
OK, if you dont have any questions close the issue

**Comment by vinnyparker** [comment]
once again thanx for the help

**Comment by hillgp** [comment]
como fazer este calculo no crudbooster formula'=>'[reajuste]+[ValorRealizado] somar estes dois valores pois quando faço a subtração funciona.

**Comment by renzocastillo** [comment]
Hi im trying to save an input field type money with decimal '2' at dec_point '.'  but when i save an e.g: 0.56 the controller saves at db 56.00.

I have done all of this as exposed. Any solution to this?

**Comment by hillgp** [comment]
veja se isso relove !!

            # START COLUMNS DO NOT REMOVE THIS LINE
            $this->col = [];
            $this->col[] = ["label"=>"Order Date","name"=>"created_at"];
            $this->col[] = ["label"=>"Order Number","name"=>"order_number"];
            $this->col[] = ["label"=>"Customer","name"=>"customers_id",
"join"=>"customers,name"];
            $this->col[] = ["label"=>"Total","name"=>"total","callback_php"
=>'number_format($row->total)'];
            $this->col[] = ["label"=>"Discount","name"=>"discount"];
            $this->col[] = ["label"=>"Tax","name"=>"tax","callback_php"=>
'number_format($row->discount)'];
            $this->col[] = ["label"=>"Grand Total","name"=>"grand_total",
"callback_php"=>'number_format($row->grand_total)'];
            # END COLUMNS DO NOT REMOVE THIS LINE

            $order_number = DB::table('orders')->max('id') + 1;
            $order_number = str_pad($order_number, 5, 0 , STR_PAD_LEFT);

            # START FORM DO NOT REMOVE THIS LINE
            $this->form = [];
            $this->form[] = ['label'=>'Customer','name'=>'customers_id',
'type'=>'select2','validation'=>'required|integer|min:0','width'=>
'col-sm-10','datatable'=>'customers,name','datatable_format'=>"name,' -
',phone"];
            $this->form[] = ['label'=>'Order Number','name'=>'order_number',
'type'=>'text','validation'=>'required|min:1|max:255','width'=>'col-sm-10',
'value'=>$order_number,'readonly'=>true];


            $columns = [];
            $columns[] = ['label'=>'Product','name'=>'products_id','type'=>
'datamodal','datamodal_table'=>'products','datamodal_columns'=>
'name,sku,sell_price,stock','datamodal_select_to'=>
'sell_price:products_price,sku:products_sku','required'=>true];
            $columns[] = ['label'=>'Product SKU','name'=>'products_sku',
'type'=>'text','readonly'=>true,'required'=>true];
            $columns[] = ['label'=>'Product Price','name'=>'products_price',
'type'=>'number','readonly'=>true,'required'=>true];
            $columns[] = ['label'=>'Quantity','name'=>'quantity','type'=>
'number','required'=>true];
            $columns[] = ['label'=>'Sub Total','name'=>'sub_total','type'=>
'number','formula'=>'[products_price] * [quantity]','readonly'=>true,
'required'=>true];
            $this->form[] = ['label'=>'Order Detail','name'=>'order_detail',
'type'=>'child','columns'=>$columns,'table'=>'orders_detail','foreign_key'=>
'orders_id'];

            $this->form[] = ['label'=>'Total','name'=>'total','type'=>
'number','validation'=>'required|integer|min:0','width'=>'col-sm-10',
'readonly'=>'true','value'=>0];
            $this->form[] = ['label'=>'Tax','name'=>'tax','type'=>'number',
'validation'=>'required|integer|min:0','width'=>'col-sm-10','value'=>0];
            $this->form[] = ['label'=>'Discount','name'=>'discount','type'=>
'number','validation'=>'required|integer|min:0','width'=>'col-sm-10','value'
=>0];
            $this->form[] = ['label'=>'Grand Total','name'=>'grand_total',
'type'=>'number','validation'=>'required|integer|min:0','width'=>'col-sm-10'
,'readonly'=>true,'value'=>0];
            # END FORM DO NOT REMOVE THIS LINE


2017-11-27 13:42 GMT-03:00 renzocastillo <notifications@github.com>:

**Comment by hillgp** [comment]
http://paunch.co/video/n0Tpl9nTqws


2017-11-28 8:25 GMT-03:00 Hill Cesar <hillpires@gmail.com>:

**Comment by renzocastilloc** [comment]
Basically  I solved it with attributes 'step'=>0.05,'type'=>'number'  at $this->form[]
